supermarq-benchmarks/supermarq/qcvv/xeb.py

```diff
@@ -215,6 +238,8 @@ def plot_speckle(self) -> None:
         fig.colorbar(
             mpl.cm.ScalarMappable(norm, cmap), ax=axs, orientation="vertical", label="Probability"
         )
+        if filename is not None:
+            fig.savefig(filename, bbox_inches="tight")
```

Comment:
for consistency should probably return `fig` here as well
```suggestion
            fig.savefig(filename, bbox_inches="tight")

        return fig
```